### PR TITLE
Clean commands and adapt new proposal API

### PR DESCRIPTION
**Pull request by clearloop**
## Desc

+ [x] Remove outdated commands
+ [x] Add last confirmed block when a proposal starts
+ [x] Correct the last leaf of the mmr proof

### CODE REVIEW

**Comment by HackFisher** [comment]
Need testing and clean api in upcoming PRs, refer https://github.com/darwinia-network/dj/issues/64

File: src/api/api.ts [hunk 1]
```diff
@@ -224,25 +185,29 @@ export class API {
     /**
      * Approve block in relayer game
      */
-    public async approveBlock(block: number, root = false): Promise<ExResult> {
+    public async approveBlock(block: number, perms = 4): Promise<ExResult> {
```
Comment (HackFisher) [inline src/api/api.ts:188]: What are the meaning of these magic numbers?

Comment (freehere107) [reply src/api/api.ts:188]: Maybe can use Enum to avoid hardcode

Comment (clearloop) [reply src/api/api.ts:188]: ### Permissions

+ read - 4
+ write - 2
+ execute - 1

This is elegant

**Pull request merged**